src/plugins/auto/multi_schedule.cpp

```diff
@@ -13,6 +13,7 @@ namespace MultiDevicePlugin {
 thread_local WorkerInferRequest* MultiSchedule::_thisWorkerInferRequest = nullptr;
 // TODO: revert to the plain variable (see header file), when we moved to the next CentOS 8.x in our support matrix
 thread_local const char* MultiSchedule::_thisPreferredDeviceName = "";
+thread_local std::vector<SoExecNetwork> MultiSchedule::_passthroughHolder = {};
```

Comment:
when user call compile_modle(network, Multi:GPU) two times in same thread, there will be two _passthroughExeNet in _passthroughHolder. if user destroy one of them, the _passthroughExeNet is not free.